Formula/wellington.rb

```diff
@@ -1,13 +1,11 @@
 require "language/go"
 
 class Wellington < Formula
-  desc "Adds file awareness to SASS"
-  homepage "https://github.com/wellington/wellington"
-
-  stable do
-    url "https://github.com/wellington/wellington/archive/v0.9.3.tar.gz"
-    sha256 "108e5626dad9494a1de7d6241a2f96c6fa5bd774133a00c301d42abd1089f3e2"
-  end
+  desc "Project-focused tool to manage Sass and spriting"
+  homepage "http://getwt.io/"
```

Comment:
you may want to look into https://github.com/certbot/certbot (i.e., Let's Encrypt)